<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicOpOutputWizardPageForm</class>
 <widget class="QWizardPage" name="BasicOpOutputWizardPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="m_widgetOutput" native="true">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QGroupBox" name="m_operationGroupBox">
            <property name="minimumSize">
             <size>
              <width>275</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Operation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QRadioButton" name="m_convexHullRadioButton">
               <property name="text">
                <string>Convex Hull</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="m_centroidRadioButton">
               <property name="text">
                <string>Centroid</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="m_mbrRadioButton">
               <property name="text">
                <string>Minimum Bounding Rectangle (MBR)</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="m_areaCheckBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Area</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="m_lineCheckBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Line</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="m_perimeterCheckBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Perimeter</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Output</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="2" column="0">
              <widget class="QLineEdit" name="m_newLayerNameLineEdit">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="1" column="2">
                <widget class="QToolButton" name="m_targetDatasourceToolButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>BD</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Repository:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLineEdit" name="m_repositoryLineEdit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QToolButton" name="m_targetFileToolButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Layer Name:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
